.github/workflows/ci.yml

```diff
@@ -27,8 +27,22 @@ jobs:
       - name: Checking if diff is empty
         run: if [ -s clang-format-diff.diff ]; then cat clang-format-diff.diff; exit 1; fi
 
+codespell:
```

Comment:
1. Indentation looks wrong. 
2. Rename the workflow to smth more common, e.g. just oneDPL CI

Reply:
Updated